JavaFXUserInterface/src/main/java/external/ModelDAOImpl.java

```diff
@@ -19,10 +26,11 @@
 public class ModelDAOImpl extends DAO implements ModelDAO {
     private static final String GET_MODEL_FROM_ASSET_TYPE_ID = "SELECT * from trained_model, model WHERE trained_model.model_id = model.model_id AND trained_model.asset_type_id = ?";
     private static final String GET_ALL_MODELS = "SELECT * from model";
-    private static final String GET_MODEL_EVALUATION = "SELECT rmse FROM model_evaluation WHERE model_id = ? AND asset_type_id = ?";
-    private static final String INSERT_RMSE = "REPLACE INTO model_evaluation SET rmse = ?,model_id = ?, asset_type_id = ? ";
+    private static final String GET_LATEST_RMSE = "select rmse from trained_model where model_id=? and asset_type_id=? and status_id=2";
```

Comment:
It would be good to have SQL keywords in caps as it is our convention

Reply:
@najimgh I updated the queries